core/templates/newspapers.html

```diff
@@ -25,6 +25,7 @@
         <thead>
         <tr>
             <th class="headerSortDown left_no_border">State</th>
```

Comment:
I think that would work except for sorting purposes.  I'm trying to think about how this would work, because the whole table is iterating over states and then over titles within the state.  I guess they would still show up in their own state but it wouldn't be immediately clear how it was working.